sendgrid/helpers/mail/asm.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, group_id=None, groups_to_display=None):
             self._group_id = group_id
```

Comment:
I actually came across this (and a couple other instances) over here too: https://github.com/sendgrid/sendgrid-python/pull/484#pullrequestreview-73282733

```python
# Shouldn't this...
self._group_id = group_id

# be this?
self.group_id = group_id
```

Reply:
My goal here was to fix tests on the master branch -- we should probably address the wider question of dog-fooding setters inside init (I agree with you) after master gets green.

Reply:
Lettuce means that the master failure was caused by a typo, of which two others exist without unit tests to fail. It might be nice to fix them in this PR since they're the "same" problem.